_rules/image-no-text-0va7u6.md

```diff
@@ -45,8 +45,7 @@ Each test target has no [visible][] [text][human language], except if at least o
 
 ## Assumptions
 
-- There is no mechanism to change the rendered text in the image resource. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] might be satisfied.
-- The specific presentation of the text rendered in the image resource can be achieved through formatted text. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] might be satisfied.
+- This rule assumes that there is no mechanism to change the rendering of text in image resources on the page, and that with CSS any requires presentation is achievable. For pages where this is not the case this rule can fail, even if [success criterion 1.4.5][sc1.4.5] may be satisfied.
```

Comment:
```suggestion
- This rule assumes that there is no mechanism to change the rendering of text within image resources on the page. For pages that *do* provide such a mechanism, this rule might fail even if [SC 1.4.5 Images of Text][sc1.4.5] is satisfied.
```

I'm not sure the "presentation can be achieved" bit is actually necessary to list as an assumption at all - isn't that covered by the `essential` definition?